contrib/brl/bbas/brad/brad_spectral_angle_mapper.cxx

```diff
@@ -351,7 +351,7 @@ void brad_spectral_angle_mapper::add_aster_dir(const std::string aster_dir) {
     std::ifstream is(fi());     // open file
     char c;
```

Comment:
char can be signed or unsigned based on the platform.  I'm OK with the simple way of silencing the warning, but does this test need to deal with different char ranges?

Reply:
Good question, I'm not sure (I'm not familiar with this code).  @andyneff any idea what exactly constitutes an "invalid character" in an aster file?